packages/jss/src/index.js

```diff
@@ -19,6 +19,11 @@ export {default as getDynamicStyles} from './utils/getDynamicStyles'
  */
 export {default as toCssValue} from './utils/toCssValue'
 
+/**
+ * Create a rule instance.
+ */
+export {default as createRule} from './utils/createRule'
```

Comment:
Its a public interface now, I wonder if we should document it

Reply:
Could you fix that? I just don't know what it is.